### DISCUSSION TITLE
docs/v1.12.x/guide/eip-snat/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#3201] [Announcements]
# docs/v1.12.x/guide/eip-snat/

https://kubeovn.github.io/docs/v1.12.x/guide/eip-snat/

**Comment by wenjunGU** [comment]
ovn-external-gw-config，这个东西我加上之后，external-gw-nodes指定的主机直接挂了，只要我启动kubelet就一直挂，什么鬼

**Reply by zbb88888** [reply]
你需要看下哪个服务挂了，以及关键 err log 是什么？

**Reply by wenjunGU** [reply]
kube-ovn-controller以及ovn-central起不来，external-gw-nodes我指到master节点了，挂的就是master。。。。

**Reply by zbb88888** [reply]
你提个 issue，详细贴下操作步骤和 log 内容吧
